src/o365/spo/commands/userprofile/userprofile-get.ts

```diff
@@ -0,0 +1,83 @@
+import { ContextInfo } from '../../spo';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+interface CommandArgs {
+  options: Options;
+}
+interface Options extends GlobalOptions {
+  userName: string;
+}
+class SpoUserProfileGetCommand extends SpoCommand {
+  public get name(): string {
+    return commands.USERPROFILE_GET;
+  }
+  public get description(): string {
+    return 'Sets user profile property for a SharePoint user';
+  }
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let spoUrl: string = '';
+    this
+      .getSpoUrl(cmd, this.debug)
+      .then((_spoUrl: string): Promise<ContextInfo> => {
+        spoUrl = _spoUrl;
+        return this.getRequestDigest(spoUrl);
+      })
+      .then((response: ContextInfo): Promise<string> => {
+        let uName : string = `i:0#.f|membership|${args.options.userName}`
+        const requestOptions: any = {
+          url: `${spoUrl}/_api/SP.UserProfiles.PeopleManager/GetPropertiesFor(accountName=@v)?@v='${encodeURIComponent(`${uName}`)}'`,
+          headers: {
+            "Accept": "application/json;odata=nometadata",
+            "Content-type": "application/json;odata=verbose",
+            "X-RequestDigest": response.FormDigestValue
+         },
+         json:true
+        };
+        return request.get(requestOptions);
+      })
+      .then((res: any): void => {
+        if (this.debug) {
+          cmd.log(vorpal.chalk.green('DONE'));
+        }
+        cmd.log(res);
```

Comment:
Hey @dips365, following our discussion @ https://github.com/pnp/cli-microsoft365/issues/1670#issuecomment-657103417 we agreed to show UserProfileProperties only in JSON output mode. Right now, they're included in the regular text output as well